main.go

```diff
@@ -300,3 +322,11 @@ func deleteTempFiles(directory string) error {
 	}
 	return nil
 }
+
+func diskSpaceCheck(mb uint64) (bool, error) {
```

Comment:
In general, function and method names should say what they do using a verb+noun structure. In this case `checkDiskSpace" would be better name.